### DISCUSSION TITLE
Support ES6 modules

### DISCUSSION THREAD

**Discussion by michail-vestnik** [uNmAnNeR/imaskjs#616] [Q&A]
Здравствуйте!

В документации написано: _For modern browsers es201X builds available (imask.es.js and imask.es.min.js)._
Но не в репозитории ни в CDN нет ссылок или готовых сборок.
Подскажите пожалуйста где можно скачать imask.es.min.js ?

Спасибо!

**Comment by uNmAnNeR** [comment]
привет. es-сборки обычно используются компиляторами в автоматическом режиме и устанавливаются через npm.
в es-сборке нет такого файла `imask.es.min.js`, потому что смысла es объединять в один файл, сжимать, выкладывать на cdn нет. Но в cdn он кстати есть, вот index:
https://unpkg.com/imask@6.4.2/esm/index.js

**Reply by uNmAnNeR** [reply]
да, я цитату проглядел. Поправлю в доке, спасибо за замечание
